pkg/handler/handler.go

```diff
@@ -479,6 +488,26 @@ func (m *Modifier) buildPodPatchConfig(pod *corev1.Pod) *podPatchConfig {
 	return nil
 }
 
+func addJitter(val int64, jitterPercent int64, min int64, max int64) (int64, error) {
```

Comment:
I think this can be just simplified to a randomization between min and max, we don't need the val. And jitterPercentage and (min, max) is kinda similar, just to add randomization